packages/ra-ui-materialui/src/list/BulkActions.js

```diff
@@ -101,26 +102,32 @@ class BulkActions extends Component {
                 >
                     <MoreVertIcon />
                 </Button>
-                <Menu
+                <Popover
```

Comment:
The popover used to appear on top of the bulk actions button

Reply:
As Menu passes its props down to Popover, isn't this bug fixed just with the `anchorOrigin` prop ?

Reply:
probably. I just used the same markup as for the Filters dropdown, to make them consistent.

But now I see that the `Poppovers` demo was removed from the mui doc, so I'm wondering if your way isn't better, and we should change the Filters dropdown instead... What do you think?

Reply:
I really don't know. I think the Popover is a low level component though. Maybe we should ask on the mui repository or ping Olivier ?

Reply:
I discovered 2 things:

- `Menu` uses `Popover` and `MenuItem` internally. To avoid repetition, it's better that we use `Menu`
- Material design explains that menus *should* open above the button, not under. The rationale is that the finger should not have to move to select the default menu item. So I removed the position setting in both menus (bulk actions, filters) to have them show up as material design says.